meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -3,16 +3,26 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI_append_lmp-wayland = " \
     file://utilities-terminal.png \
     file://background.jpg \
+    file://weston.env \
+    file://weston.service.patch \
 "
 
 FILES_${PN}_append_lmp-wayland = " \
     ${datadir}/weston \
 "
 
+INI_UNCOMMENT_ASSIGNMENTS = " \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'x11 wayland', 'xwayland=true', '', d)} \
+"
+
 do_install_append_lmp-wayland() {
     install -d ${D}${datadir}/weston/backgrounds
     install -d ${D}${datadir}/weston/icon
 
     install -m 0644 ${WORKDIR}/utilities-terminal.png ${D}${datadir}/weston/icon/utilities-terminal.png
     install -m 0644 ${WORKDIR}/background.jpg ${D}${datadir}/weston/backgrounds/background.jpg
-}
+
+    for assignment in ${INI_UNCOMMENT_ASSIGNMENTS}; do
+        uncomment "$assignment" ${D}${sysconfdir}/xdg/weston/weston.ini
```

Comment:
The function uncomment is only defined in the bbappend from meta-freescale.

To test this, try disabling the meta-freescale layer when generating the image.

As we need the OPTARGS changes for getting rpi (and other machines to work), mind splitting this into 2 patches, one for OPTARGS and another for INI_UNCOMMENT_ASSIGNMENTS?